### PR TITLE
Adding custom footer feature for GCIntranet footer

### PR DESCRIPTION
**Pull request by ahmad-shahid**

### CODE REVIEW

File: test/specs/footer.js [hunk 1]
```diff
@@ -129,15 +129,15 @@ describe('Footer section tests for GCIntranet', () => {
         refTop: '{"cdnEnv": "localhost"}',
         top: '{"cdnEnv": "localhost", "siteMenu": false}',
         preFooter: '{"cdnEnv": "localhost"}',
-        footer: '{"showFeatures": true, "cdnEnv": "localhost"}',
+        footer: '{"showFeatures": true, "footerPath": "https://esdc.prv/_conf/assets/en/footer/esdcfooter-eng.html", "cdnEnv": "localhost"}',
```
Comment (plup2) [inline test/specs/footer.js:132]: The tests shouldn't rely on external resources where possible, especially ones that can change without us knowing.

I see two options:
- If we actually have a similar file in the CDTS content, we could have a relative path to it
- OR I'm thinking we could create a folder like `test/html/global` or `test/html/static` or something like that and put the files needed for testing

File: test/specs/footer.js [hunk 2]
```diff
@@ -161,6 +161,11 @@ describe('Footer section tests for GCIntranet', () => {
         await defaultFooterLinksIntranet(theme, 'fr');
     });
 
+    it('Validate custom footer', async () => {
```
Comment (plup2) [inline test/specs/footer.js:164]: Just a question, the test prior to this one (ie `it('Validate the default footer links'`) , it tests that everything is ok when there is no custom footer, right?  (in other words, we have such a test, right?)

**Review by plup2 — CHANGES_REQUESTED** [review]

File: test/specs/footer.js (hunk 1 above)
Comment (ahmad-shahid) [reply test/specs/footer.js]: fixed! pointing to the footer file in the global folder now

File: test/specs/footer.js (hunk 2 above)
Comment (ahmad-shahid) [reply test/specs/footer.js:164]: Yup, that test checks the links for the default footer

**Review by plup2 — APPROVED** [review]

**Pull request merged**